maintainers/maintainer-list.nix

```diff
@@ -5159,6 +5159,11 @@
     github = "ThomasMader";
     name = "Thomas Mader";
   };
+  thomasjm = {
```

Comment:
Alphabetically, `thomasjm` should be before `ThomasMader`, I think.

Reply:
Oh yeah, I was thinking uppercase comes first but that's not how this file works. Fixed.